curvefs/src/mds/fs_manager.cpp

```diff
@@ -308,10 +308,17 @@ FSStatusCode FsManager::CreateFs(const ::curvefs::mds::CreateFsRequest* request,
         option_.s3AdapterOption.s3Address = s3Info.endpoint();
         option_.s3AdapterOption.bucketName = s3Info.bucketname();
         s3Adapter_->Reinit(option_.s3AdapterOption);
-        if (!s3Adapter_->BucketExist()) {
+        const Aws::String aws_key(fsName.c_str(), fsName.size());
```

Comment:
1. Current modifications are not universal, you can add a function named like 'BucketExistCheckPut' and put a random content(maybe time-dependent) in it.
2. This requires two interface calls. Consider whether you can use the head bucket to judge first, and if it fails (all failures or feature error codes), then make a put judgment?

Reply:
1: Current modifications doesn't cover the condition that if the bucket doesn't exist, we should create bucket 
automaticlly.
2: I think it's not necessary to call head bucket, When s3 providers disable the interface, we need to call put object, 
When having the permission to call interface head bucket, we need also call put object to make sure the fs can upload object normally.

Reply:
fixed